src/pipecat/services/openai/realtime/llm.py

```diff
@@ -819,11 +861,40 @@ def create_context_aggregator(
             the user and one for the assistant, encapsulated in an
             OpenAIContextAggregatorPair.
         """
-        context.set_llm_adapter(self.get_llm_adapter())
-
-        OpenAIRealtimeLLMContext.upgrade_to_realtime(context)
-        user = OpenAIRealtimeUserContextAggregator(context, params=user_params)
+        # Log warning about transcription frame direction change in 0.0.92.
```

Comment:
Note to self: if the changes in this PR don't land in 0.0.92, make sure to update all references to 0.0.92.